workshops/workshops.js

```diff
@@ -0,0 +1,40 @@
+import { checkAuth, deleteParticipant, getUser, getWorkshops } from '../fetch-utils.js';
+import { renderParticipant, renderWorkshop } from '../render-utils.js';
+
+const loginEmail = document.getElementById('login-email');
+const workshopsEl = document.getElementById('workshops');
+const addButton = document.getElementById('add-participant');
+
+window.addEventListener('load', async () => {
+    checkAuth();
+    loginEmail.textContent = getUser().email;
+
+    await displayWorkshops();
+});
+
+async function displayWorkshops() {
+    workshopsEl.textContent = '';
+    const workshops = await getWorkshops();
+    console.log(workshops);
```

Comment:
quick reminder to get rid of unnecessary console logs when you do your final push up of code. It's not overly disruptive of the app but when you get into the job field companies will not want these in final code.